modern/src/map/GeofenceEditMap.js

```diff
@@ -1,24 +1,559 @@
-import '@mapbox/mapbox-gl-draw/dist/mapbox-gl-draw.css'
-import MapboxDraw from '@mapbox/mapbox-gl-draw';
-import { useEffect } from 'react';
-
+import * as React from 'react';
+import { MapContext } from '@urbica/react-map-gl';
+import MapboxDraw from 'C:/Users/ALTERNATOR/Desktop/Rapport-PFE/PFE/Traccar/nextTrac/modern/node_modules/@mapbox/mapbox-gl-draw';
+import theme from '@mapbox/mapbox-gl-draw/src/lib/theme';
 import { map } from './Map';
+import 'mapbox-gl/dist/mapbox-gl.css';
+import '@mapbox/mapbox-gl-draw/dist/mapbox-gl-draw.css';
+import {useState} from 'react';
```

Comment:
```suggestion
import { useState } from 'react';
```

Reply:
it s working i just left those comments there, i will fomat it now